src/webots/vrml/WbProtoManager.cpp

```diff
@@ -145,7 +145,7 @@ WbProtoModel *WbProtoManager::findModel(const QString &modelName, const QString
     const WbVersion protoVersion = checkProtoVersion(parentFilePath, &foundProtoVersion);
     if (foundProtoVersion && protoVersion < WbVersion(2022, 1, 0)) {
       const QString backwardsCompatibilityMessage =
-        tr("Please adapt your project to R2022b following these instructions: "
+        tr("Please adapt your project to R2023a following these instructions: "
```

Comment:
I was not sure about it, I think that it feels strange if you open R2023a and it says: adapt to R2022b

Reply:
I think it would be odd if they install say R2024 and the warning tells them to update the world to R2022. So having the message warn them to adapt the world to the latest version I think makes sense. It raises a good point though, namely that the script currently converts worlds prior to R2022b to R2022b, as such if we keep R2023a here, we should probably adapt that script too or it won't do it in practice

Reply:
I updated the script to convert to R2023a